crates/bws/src/main.rs

```diff
@@ -320,12 +323,20 @@ async fn process_commands() -> Result<()> {
         })
         .transpose()?;
 
+    let state_file_path = state::get_state_file_path(
+        profile.and_then(|p| p.state_file_dir).map(Into::into),
+        access_token_obj.access_token_id.to_string(),
+    );
```

Comment:
Do we have a way for not creating state files?

Reply:
Changed `bws` so to use state you have to specify `state_file_dir` in the `~/.bws/config` file, rather than then using `~/.bws/state` by default.

Resolved in [ce50cf1](https://github.com/bitwarden/sdk/pull/388/commits/ce50cf1a3cfa405449b8dbb8416c3007c98540ce)